src/pages/MainPage/components/FavoriteMovieBlock/components/OneFavoriteMovie/index.tsx

```diff
@@ -0,0 +1,105 @@
+import Button from "@components/Button";
+import { FavoriteMovie } from "../..";
+import { X, Eye } from "lucide-react";
+import NameOfGenres from "../NameOfGenres";
+import { useLingui } from "@lingui/react/macro";
+import { Genre } from "@services/tmdbQuery";
+
+interface OneFavoriteMovieProps {
+  favMovie: FavoriteMovie;
+  viewMode: "grid" | "list";
+  toggleWatchedStatus: (id: number) => void;
+  deleteHandleClick: (id: number) => void;
+  genres: Genre[];
+  number: string;
+}
+
+const API_PICS = import.meta.env.VITE_PICS_URL;
+
+function OneFavoriteMovie({
+  favMovie,
+  viewMode,
+  toggleWatchedStatus,
+  deleteHandleClick,
+  genres,
+  number,
+}: OneFavoriteMovieProps) {
+  const { t } = useLingui();
+
+  return viewMode === "list" ? (
+    <div
+      key={favMovie.id}
+      className={`${
+        favMovie.watchedStatus ? "grayscale opacity-50" : ""
+      } flex flex-row items-center gap-4 justify-between border border-gray-400 rounded-lg bg-gray-100 px-2`}
+    >
+      <span>{number}</span>
+      <p className="break-words overflow-hidden">{favMovie.title}</p>
+      <img
+        className="my-2 max-h-64"
+        alt={`poster of ${favMovie.title}`}
+        src={API_PICS + favMovie.posterPath}
+      />
+      <span>{t`Primary release year: ${favMovie.releaseDate.slice(
+        0,
+        4
+      )}`}</span>
+      <div className="flex flex-col">
```

Comment:
```suggestion
      <div className="flex flex-col">
```
можно убрать в NameOfGenres раз это повторяется и ниже